frontend/src/pages/operation_show/layout/sidebar/stylesheet.styl

```diff
@@ -65,6 +65,55 @@
     opacity: 1
     background-color: $background
 
+.favorite
```

Comment:
same note here with the css change (though I didn't test this one out)

Reply:
changed this as well